src/main/java/io/turntabl/employementprofilingsystem/Controllers/RequestController.java

```diff
@@ -31,49 +45,147 @@ public void makeARequest(@RequestBody RequestTO request) {
         String reportDate = DateFor.format(request.getRequest_report_date());
 
         try {
-            HolidayRequestMail.requestMessage("isaac.agyen@turntabl.io", request.getFrom() ,"Holiday request", startDate, reportDate, request.getRequester_name());
+            ApproverMail.requestMessage(System.getenv("APPROVER_EMAIL"),request.getFrom(), "Holiday request", startDate, reportDate);
         } catch (IOException e) {
             e.printStackTrace();
         } catch (GeneralSecurityException e) {
             e.printStackTrace();
         }
     }
```

Comment:
are you sure that is all the exceptions to catch,
You can catch General exception too